src/raven/client.clj

```diff
@@ -230,24 +233,37 @@
   [context]
   (or (:http context) (http/build-client {})))
 
+(defn perform-in-memory-request
+  "Perform an in-memory pseudo-request, actually just storing the payload on a storage
+    atom, to let users inspect/retrieve the payload in their tests."
+  [payload]
+  (swap! http-requests-payload-stub conj payload))
+
+(defn perform-http-request
+  [context dsn ts payload]
+  (let [json-payload  (json/generate-string payload)
+        {:keys [key secret uri pid]} (parse-dsn dsn)
+        sig                          (sign json-payload ts key secret)]
+    (http/request (get-http-client context)
+                  {:uri            (format "%s/api/store/" uri pid)
```

Comment:
Is it missing a `%s`, `%d` ?

Reply:
Ah, good catch. pid should not be passed at all it seems.

Fixed.